Dockerfile

```diff
@@ -5,4 +5,6 @@ RUN hugo
 
 FROM nginx:alpine
 RUN rm -rf /usr/share/nginx/html/*
-COPY --from=builder /site/public /usr/share/nginx/html
\ No newline at end of file
+COPY nginx.conf /etc/nginx/config.d/jb-404.conf
+RUN echo "\ninclude /etc/nginx/config.d/*.conf;" > /etc/nginx/nginx.conf
```

Comment:
Since we're leveraging the default config file in the nginx container ( default config uploaded [here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/files/9651974/default.txt), typically a `.conf` extension but GH doesn't like that :sweat_smile: ), we can just un-comment the 404 line in the default config file.

```suggestion
RUN sed -i 's/#error_page/error_page/' /etc/nginx/conf.d/default.conf
```

Then we don't need the `nginx.conf` file at all as well.

### extra details

For those interested, I looked at the default container by shelling into the alpine container with this command: `docker container run --rm -it --entrypoint=sh --name jbcom nginx:alpine`. I copied it out of the container after starting it up (with the shell command) and then ran: `docker cp jbcom:/etc/nginx/conf.d/default.conf .`

Then I merged this PR (branch [here](https://github.com/elreydetoda/jupiterbroadcasting.com/tree/patch/modifying_e2e_GH_action-CGBassPlayer-feature/404-page)) into my E2E one (#353), because I added a way to easily build the prod container by running this: `docker build --build-arg='BASE_URL=TESTS_WEB' --rm -t jb-jbcom:latest '.'`. Which then let me view it locally with this: `docker container run --rm -it -p 1313:80 --name jbcom jb-jbcom:latest` (after I've already finished being in a shell for the other container).

Reply:
I appreciate the feedback on this @elreydetoda! I am not familiar with the deployment side of things (but I love learning about it) so this was more or less taking a guess at how to tackle this.

Reply:
No problem, you did awesome figuring out how to setup server blocks in nginx 🥳

Awesome! That is why I included a summary of how I figured it out, I'd hoped at least someone was curious 😅 (if not then it was at minimum a note to myself 🙃)